tests/test_s3_ndjson_format.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+import json
+import boto3
+from botocore.exceptions import NoCredentialsError
+
+
+class TestS3NDJSONFormat(unittest.TestCase):
```

Comment:
don't extend unittest.TestCase, if it requires external integration

Reply:
done